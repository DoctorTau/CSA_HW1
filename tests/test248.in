448
-3344 -1941 -2934 -8456 -8653 1950 8649 -2192 4205 2171 -8168 3804 2146 6305 7766 -8653 -7376 -9752 -4289 -4561 -6397 1457 -8148 825 7550 8988 7355 2877 335 -9551 -7571 -8391 -8126 -634 2337 8669 8284 -760 -9282 -6864 4807 6045 -1834 -8779 -7966 -2019 6538 -2996 3273 5096 -8195 7489 -1525 4972 4738 -2641 7340 77 8265 5627 5415 -2767 5678 1340 -5654 -5726 4879 -2226 -9730 9960 -8016 -5830 702 2405 -9620 -7007 -7812 3579 -6584 -2394 -7864 -3777 3911 5018 8673 -3090 758 2127 -149 7786 -7358 1316 7590 -6204 7508 1601 6944 -7268 -428 -9770 -9817 3741 -2224 5142 491 3987 6706 1290 -5278 -1680 432 212 5136 8768 -5425 -8337 -372 -577 6346 6008 8155 2578 -4864 965 7823 -8233 -8431 -1098 8412 8122 -8672 -4484 -5417 5343 155 855 -1931 6300 4950 -2873 -5026 2132 84 -3434 -9540 -38 2106 5633 9602 -9678 7461 8910 629 7351 6690 2443 9859 8085 -4513 -3104 -3099 5846 4270 8492 4528 -4471 -8157 8212 -5907 878 -9306 -6758 9420 9493 -8236 4618 -2144 9714 -6241 1426 3776 4438 8325 1245 -4547 9416 -9736 9188 5042 3661 6656 2235 7466 -9084 -5002 8255 -7544 -2841 7901 6022 -7154 -3156 4824 -5603 -883 3792 -4077 -3227 6303 4048 2586 6914 9128 3507 -8330 8508 -9211 -5864 -5507 8791 6670 -5699 -8130 -5059 791 3515 -9781 5386 -5296 -9985 4955 -392 -7333 423 4525 -1566 -8161 8175 4034 2952 -3209 8672 8837 7059 -7055 -2804 -3256 -7956 2052 -3430 8785 -3120 5119 -161 5590 -1332 6763 -878 -7736 3755 -8475 -1169 -8813 -3114 873 -2004 6340 -2862 9213 -6588 -7309 -4018 7093 -9337 3509 -4532 3796 4356 9862 -1875 -5596 -8249 -4093 -9185 -5539 -2989 6382 1832 8851 -6559 -2726 -4521 -6122 -7074 -3595 1655 -4074 -1643 8745 9586 1597 9511 -4827 -4448 2077 8230 2633 -2727 6141 2536 7211 -9031 -7043 -1178 -127 2045 5060 -756 -1469 -254 4657 7120 2112 -1693 5581 2270 6081 9901 7355 -804 -3590 4856 3539 -4523 -8239 -1572 -7543 -3756 -5135 4753 7420 4756 -2430 3930 9664 6533 7721 -6278 9482 -1463 5469 -3588 -4114 -8602 -5488 -6713 -5308 -5434 -3178 7860 -8030 9382 -9791 9341 9973 572 4847 -7948 9307 -7580 9447 -5852 -2348 -9063 3926 -9319 -8836 4017 -6029 -5911 2398 2499 3839 8248 7127 6997 1262 9197 -3149 9871 3323 5447 -8662 1136 4318 -8710 1767 4551 9435 -9959 -1528 799 2695 7589 -1851 -821 5323 -8765 6553 -5293 1287 3242 4082 5235 5672 -4999 -2693 8849 6288 5229 706 798 3271 7080 -272 -1678 -1459 2743 -375 430 -415 2398 -5625 1642 2218 -4871 -1894 -1066 -7138 -6807 -2813 -6049 -9571 -5866 -9139 2833 -998 -9324 