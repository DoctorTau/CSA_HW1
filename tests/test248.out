-3344 -2934 -8456 -2192 -8168 -7376 -9752 -8148 -8126 -634 -760 -9282 -6864 -1834 -7966 -2996 -5654 -5726 -2226 -9730 -8016 -5830 -9620 -7812 -6584 -2394 -7864 -3090 -7358 -6204 -7268 -428 -9770 -2224 -5278 -1680 -372 -4864 -1098 -8672 -4484 -5026 -3434 -9540 -38 -9678 -3104 -9306 -6758 -8236 -2144 -9736 -9084 -5002 -7544 -7154 -3156 -8330 -5864 -8130 -5296 -392 -1566 -2804 -3256 -7956 -3430 -3120 -1332 -878 -7736 -3114 -2004 -2862 -6588 -4018 -4532 -5596 -2726 -6122 -7074 -4074 -4448 -1178 -756 -254 -804 -3590 -1572 -3756 -2430 -6278 -3588 -4114 -8602 -5488 -5308 -5434 -3178 -8030 -7948 -7580 -5852 -2348 -8836 -8662 -8710 -1528 -272 -1678 -1894 -1066 -7138 -5866 -998 -9324 